dali/test/python/test_operator_affine_transforms.py

```diff
@@ -258,7 +262,8 @@ def shear_affine_mat(shear=None, angles=None, center=None):
     return affine_mat
 
 
-def check_transform_shear_op(shear=None, angles=None, center=None, has_input=False, reverse_order=False, batch_size=1, num_threads=4, device_id=0):
+def check_transform_shear_op(shear=None, angles=None, center=None, has_input=False,
+                             reverse_order=False, batch_size=1, num_threads=4, device_id=0):
```

Comment:
Group transform-specific, generic and pipeline parameters
```suggestion
def check_transform_shear_op(shear=None, angles=None, center=None,
                             has_input=False, reverse_order=False,
                             batch_size=1, num_threads=4, device_id=0):
```